<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionAndAnswerSiteClass</class>
 <widget class="QMainWindow" name="QuestionAndAnswerSiteClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuestionAndAnswerSite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="questionsListWidget"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addQuestionPushButton">
        <property name="text">
         <string>Add question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newQuestionLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="answersListWidget"/>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addAnswerPushButton">
        <property name="text">
         <string>Add answer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newAnswerLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="selectedAnswerLineEdit"/>
      </item>
      <item>
       <widget class="QSpinBox" name="votesSpinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QuestionAndAnswerSite.qrc"/>
 </resources>
 <connections/>
</ui>
